TTS/tts/datasets/formatters.py

```diff
@@ -15,6 +15,14 @@
 
 def coqui(root_path, meta_file, ignored_speakers=None):
     """Interal dataset formatter."""
+    filepath = os.path.join(root_path, meta_file)
+    # ensure there are 4 columns for every line
+    with open(filepath, "r", encoding="utf8") as f:
+        lines = f.readlines()
+    for idx, line in enumerate(lines):
+        if len(line.split("|")) != 4:
```

Comment:
Better to check if all lines have the same number of columns instead of enforcing that number 4. The idea of that loader is that it can support metadata files with only audio_file and text, or also extra fields of data like comments or dates without preventing loading.